packages/main/src/Select.hbs

```diff
@@ -29,6 +28,8 @@
 		dir="{{effectiveDir}}"
 	></ui5-icon>
 
+	<span id="{{_id}}-labelText" class="ui5-hidden-text">{{ariaLabelText}} {{_text}}</span>
```

Comment:
it will be more correct if the aria-labelledby point to both the _text span and the ariaLabelText coming from the component

example aria-labelledby="{{_id}}-labelText {{_id}}-label"